kafka-client/src/main/java/dev/responsive/kafka/internal/db/mongo/MongoKVTable.java

```diff
@@ -150,21 +164,63 @@ public MongoKVFlushManager init(final int kafkaPartition) {
   }
 
   @Override
-  public byte[] get(final int kafkaPartition, final Bytes key, final long minValidTs) {
-    final KVDoc v = docs.find(Filters.and(
-        Filters.eq(KVDoc.ID, keyCodec.encode(key)),
-        Filters.gte(KVDoc.TIMESTAMP, minValidTs)
-    )).first();
-    return v == null ? null : v.getValue();
+  public byte[] get(final int kafkaPartition, final Bytes key, final long streamTimeMs) {
```

Comment:
Let's make sure to add tests for MongoDB as well. I'd actually prefer if we just threw an exception for this PR if there's anything but a static default TTL so we can have all the mongo changes be part of a future PR.